tests/vxlan/test_vxlan_ecmp.py

```diff
@@ -68,7 +68,7 @@
 
 pytestmark = [
     # This script supports any T1 topology: t1, t1-64-lag, t1-lag.
-    pytest.mark.topology("t1", "t1-64-lag", "t1-lag"),
+    pytest.mark.topology("t1"),
```

Comment:
Please revert this line, because of this PR: https://github.com/Azure/sonic-mgmt/pull/5117